js/coinbase.js

```diff
@@ -104,62 +104,84 @@ module.exports = class coinbase extends Exchange {
             },
             'api': {
                 'public': {
-                    'get': [
-                        'currencies',
-                        'time',
-                        'exchange-rates',
-                        'users/{user_id}',
-                        'prices/{symbol}/buy',
-                        'prices/{symbol}/sell',
-                        'prices/{symbol}/spot',
-                    ],
+                    'original': {
```

Comment:
@Dan-krm question, is the API share the same hostname and versioning, should we modify its structure? Do you see any disadvantage in keeping everything as it was, plus the new methods?

Reply:
I agree with carlosmiei, I think we should place it in it's own file

Reply:
The 'original' api only has transfer and withdrawal endpoints and the advanced api only has trading endpoints, they both use the same base URL except original is v2 and advanced is v3.

If we were to keep everything as it was plus the new endpoints I'm not sure how we would handle that in sign so that all of the methods can be used, but let me know if there's an alternative approach that would be better

Reply:
@Dan-krm ok so, instead of `original` and `advanced` I think we should use `v2` and `v3` in the API definition, it's a bit more clear i think

Reply:
I agree, I made the changes, let me know if there is anything else.

@carlosmiei On line 225 Currently the only purpose of setting this.options `'advanced': true,` is to avoid an exception thrown in handleErrors which I think is there because coinbase returns detailed failure responses so it can look like a users order is successful when it isn't. Not all of the advanced trade endpoints have 'data' in their response though so I'm thinking I can do error checking to check for `'success': true,` in the response of the individual v3 methods and maybe add them to handleErrors later after testing all of the new methods